.travis.yml

```diff
@@ -41,7 +41,7 @@ install:
 # --> lint the README documentation with mdl
 script:
   - shopt -s globstar
-  - pipenv run pytest gatorgrouper/tests --cov-config pytest.cov --cov
+  - pipenv run pytest tests --cov-config pytest.cov --cov
```

Comment:
Does this correctly run the gatorgrouper tests? It probably doesn't... the best option here is to let pytest discover the tests on its own, probably.

Reply:
Ah never mind, you changed the test location